plugin/src/main/java/com/denizenscript/denizen/objects/EntityTag.java

```diff
@@ -667,6 +667,10 @@ public boolean isPlayer() {
         return entity instanceof Player && !isNPC();
     }
 
+    public <T extends Entity> T as(Class<T> entityClass) {
+        return entityClass.cast(getBukkitEntity());

```

Comment:
same question as the other PR - why `Clazz.cast` instead of just `(T)`

Reply:
Changed to a direct cast